modules/ROOT/pages/sftp/sftp-documentation.adoc

```diff
@@ -68,384 +78,456 @@ An FileSystemProvider which provides instances of SftpFileSystem from instances
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  +++A TimeUnit which qualifies the #responseTimeoutUnit attribute. Defaults to SECONDS+++ |  +++SECONDS+++ | {nbsp}
-| Host a| String |  +++The FTP server host, such as www.mulesoft.com, localhost, or 192.168.0.1, etc+++ |  | *x*{nbsp}
-| Port a| Number |  +++The port number of the SFTP server to connect on+++ |  +++22+++ | {nbsp}
-| Username a| String |  +++Username for the FTP Server. Required if the server is authenticated.+++ |  | {nbsp}
-| Password a| String |  +++Password for the FTP Server. Required if the server is authenticated.+++ |  | {nbsp}
-| Passphrase a| String |  +++The passphrase (password) for the identityFile if required. Notice that this parameter is ignored if #identityFile is not provided+++ |  | {nbsp}
-| Identity File a| String |  +++An identityFile location for a PKI private key.+++ |  | {nbsp}
+** DAYS |  A TimeUnit which qualifies the *Response Timeout* attribute.  Defaults to SECONDS. |  SECONDS | 
+| Host a| String |  The FTP server host, such as `www.mulesoft.com`, localhost, or 192.168.0.1, etc. |  | x
+| Port a| Number |  The port number of the SFTP server to connect on. |  22 | 
+| Username a| String |  Username for the FTP Server. Required if the server is authenticated. |  | 
+| Password a| String |  Password for the FTP Server. Required if the server is authenticated. |  | 
+| Passphrase a| String |  The pass phrase (password) for the identityFile if required. Notice that this parameter is ignored if #identityFile is not provided. |  | 
```

Comment:
Should this be `identifyFile`?

Reply:
I think it should really be "identity file" as there's no definition of identityFile in the reference. However, #identityFile should be **Identity File** because they're then discussing the field.